config/formats.js

```diff
@@ -23,6 +23,12 @@ let Formats = [
 		team: 'random',
 		ruleset: ['PotD', 'Pokemon', 'Sleep Clause Mod', 'HP Percentage Mod', 'Cancel Mod'],
 	},
+	{
+		name: "[Gen 7] Random Battle (Blitz)",
+		mod: 'gen7',
+		team: 'random',
+		ruleset: ['PotD', 'Pokemon', 'Sleep Clause Mod', 'HP Percentage Mod', 'Cancel Mod', 'Blitz'],
```

Comment:
```suggestion
		ruleset: ['[Gen 7] Random Battle', 'Blitz'],
```

`ruleset` provides a very easy way to inherit rules from other formats, just sayin'.